libass/ass_bitmap.c

```diff
@@ -223,7 +225,9 @@ void fix_outline(Bitmap *bm_g, Bitmap *bm_o, uint8_t alpha_g)
 
     for (int32_t y = 0; y < b - t; y++) {
         for (int32_t x = 0; x < r - l; x++) {
-            int num = FFMAX(o[x] - g[x], 0) * 65025;
+            int num = blurred ?
+                o[x] * (255 - g[x]) * 255 :
```

Comment:
Good point, I can see how this can seem inconsistent. Ironically, this is precisely because I tried to make the more external instances consistent with existing external code and the inner loop consistent within itself.

The problem is that we already use `0xFF` elsewhere when we check alpha values (e. g. when the `filter.flags` are first computed), so I tried to follow suit; but inside this loop, we have arithmetic with not just 255 but also 255² = 65025, and spelling 65025 as hex isn’t pretty at all. I suppose I could write `0xFF * 0xFF`?

Reply:
Rewrote the earlier instance within this particular function as `255 - alpha_g`.